ncm-opennebula/src/main/perl/opennebula.pm

```diff
@@ -334,8 +352,11 @@ sub restart_opennebula_service {
         $srv = CAF::Service->new(['opennebula'], log => $self);
     } elsif ($service eq "sunstone") {
         $srv = CAF::Service->new(['opennebula-sunstone'], log => $self);
+    } elsif ($service eq "kvmrc") {
+        $self->info("Updated $service file. onehost sync is required.");
+        $self->sync_opennebula_hyps("-f");
```

Comment:
it forces to update VMM drivers on each hyp. Just to be sure that all hyps are updated if VMM conf is changed by quattor.